cmd/prometheus/config.go

```diff
@@ -288,6 +297,25 @@ func parseInfluxdbURL() error {
 	return nil
 }
 
+func parseAlertmanagerURL() error {
+	if cfg.alertmgrURL == "" {
+		return nil
+	}
+
+	url, err := url.Parse(cfg.alertmgrURL)
+	if err != nil {
+		return err
+	}
+
+	if url.Path == "" {
```

Comment:
Does this mean it's not possible to send alerts to http://example.com/  ?

This magic could be confusing for users.